website/docs/d/s3_account_public_access_block.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "S3 Control"
+layout: "aws"
+page_title: "AWS: aws_s3_account_public_access_block"
+description: |-
+  Provides S3 account-level Public Access Block Configuration
+---
+
+# Data Source: aws_s3_account_public_access_block
+
+The S3 account public access block data source returns account-level public access block configuration.
+
+## Example Usage
+
+```terraform
+data "aws_s3_account_public_access_block" "example" {
+}
+
+output "foo" {
+  block_public_acls_value = data.aws_s3_bucket_policy.example.block_public_acls
+  block_public_policy_value = data.aws_s3_bucket_policy.example.block_public_policy
+  ignore_public_acls_value = data.aws_s3_bucket_policy.example.ignore_public_acls
+  restrict_public_buckets_value = data.aws_s3_bucket_policy.example.restrict_public_buckets
+}
```

Comment:
`output` block is not required in the documentation

Reply:
Removed output block